src/store/actions/voting.js

```diff
@@ -85,22 +85,45 @@ export const votesSubmitted = ({ fee, votes }) =>
         ...transaction,
         moduleAssetId,
       },
+      isHwSigning: account.loginType !== loginTypes.passphrase.code,
     };
 
-    const [error, tx] = account.loginType === loginTypes.passphrase.code
-      ? await to(create(params, tokenMap.LSK.key))
-      : await to(signVoteTransaction(account, transaction));
+    let [error, tx] = await to(create(params, tokenMap.LSK.key));
 
     if (error) {
-      return dispatch({
+      dispatch({
         type: actionTypes.transactionSignError,
         data: error,
       });
+      return;
+    }
+
+    if (params.isHwSigning) {
```

Comment:
This decreases maintainability. I've explained more in `src/utils/api/transaction/lsk.js`